packages/desktop-client/src/components/reports/Header.js

```diff
@@ -129,18 +145,112 @@ function Header({ title, start, end, show1Month, allMonths, onChangeDates }) {
         <Button
           bare
           style={{ marginLeft: 15 }}
-          onClick={() => onChangeDates(...getLatestRange(12))}
+          onClick={() => onChangeDates(...getLatestRange(11))}
         >
           1 Year
         </Button>
         <Button
           bare
-          style={{ marginLeft: 15 }}
+          style={{
+            marginLeft: 15,
+            display: !showAllTime ? 'inherit' : 'none',
+          }}
           onClick={() => onChangeDates(...getFullRange(allMonths))}
         >
           All Time
         </Button>
       </View>
+      <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'flex-start',
+          paddingTop: 15,
+        }}
+      >
+        <View style={{ marginRight: 10, marginTop: 2, marginBottom: 5 }}>
+          <FilterButton onApply={onApplyFilter} disableFilter={disableFilter} />
+        </View>
+
+        {filters && filters.length > 0 && (
+          <AppliedFilters filters={filters} onDelete={onDeleteFilter} />
+        )}
+      </View>
+    </View>
+  );
+}
+
+export function TotalsTrends({ title, isElement, Chart, id, OnClick }) {
+  return (
+    <View
+      style={{
+        color: colors.n3,
+        marginLeft: 20,
+      }}
+    >
+      <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'center',
+          height: 16,
+          marginBottom: 2,
+        }}
+      >
+        <ChartItem
+          isElement={isElement}
+          id={id}
+          handleClick={OnClick}
+          Icon={Chart}
+          title={title}
+        />
+      </View>
+      <View
+        style={[
+          selectionBoxSmall.SquareShapeView,
+          { display: isElement ? 'inherit' : 'none' },
+        ]}
+      />
+    </View>
+  );
+}
+
+export function HeaderReport({
+  title,
+  id,
+  handleMouseHover,
+  handleMouseLeaveHeader,
+  onHeaderClick,
+  isElement,
+  isDefault,
+}) {
+  return (
+    <View
+      style={{
+        marginLeft: 20,
+        flexShrink: 0,
+        color: isDefault ? colors.n3 : colors.n7,
+      }}
+    >
+      <View
+        onMouseOver={handleMouseHover}
+        onMouseOut={handleMouseLeaveHeader}
+        onClick={onHeaderClick}
```

Comment:
Can you use a `<Button />` here for accessibility (and generally match this better to the `ChartItem` styles?)